### PR TITLE
Added tech stack

### LINKED ISSUE #1: Finalisation of tech stack
**Requirements**:
- [x] Tech stack for the server.
- [x] Tech stack for the parser.
- [x] Tech stack for GitHub handler.
- [x] Tech stack for UI.
- [x] Tech stack for DB.

**Definition of done**:
- [x] All the tech stacks above are finalized and added as a tech document in the repo.

### PR DESCRIPTION
**Pull request by shubhaangi04**
Added documentation of tech stack.

### CODE REVIEW

File: docs/Tech Stack.txt [hunk 1]
```diff
@@ -0,0 +1,11 @@
+1. Tech stack for the server: Springboot, NodeJS, DJango 
+	-> Springboot because tech stack for parser would be entirely Java. 
+2. Tech stack for the parser: Java, Python 
+	-> Java because tech stack for server is Spring Boot
```
Comment (krvikas1011) [inline docs/Tech Stack.txt]: Don't you think these two are circular?

File: docs/Tech Stack.txt [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+3. Tech stack for GitHub handler: Java
+4. Tech stack for UI: React, AngularJS 
+	-> ReactJS is a JavaScript library that combines the speed of JavaScript and uses a new way of rendering webpages, making them highly dynamic and responsive to user input. 
+	  Also, it is compatible with Spring Boot.
```
Comment (krvikas1011) [inline docs/Tech Stack.txt]: So is Angular.

File: docs/Tech Stack.txt [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+5. Tech stack for DB: MongoDB, MySQL
+	-> MongoDB is faster and in MongoDB, data is stored in JSON-like documents that can have varied structures. 
+	 To improve query speed, MongoDB can store related data together, which is accessed using the MongoDB query language.
```
Comment (krvikas1011) [inline docs/Tech Stack.txt]: The major consideration of selecting a DB is the form of data that we are going to store in it.

File: docs/Tech Stack.txt [hunk 4]
```diff
@@ -0,0 +1,11 @@
+1. Tech stack for the server: Springboot, NodeJS, DJango
```
Comment (krvikas1011) [inline docs/Tech Stack.txt]: I would prefer the below format, see if it looks cleaner to you:

```
Tech Stacks
-----------------------------------
-----------------------------------

Server
----------------
Selected: Spring Boot
Other options considered : Django, Node, blah, blah.....

Reason: blahhhhhh................


Parser
-----------------
Selected: ...........
.....
.....

```

**Review by thisissvikas — CHANGES_REQUESTED** [review]

**Comment by krvikas1011** [comment]
A couple of more suggestions, I would like to add:

1. Make sure you add a description to every PR that you raise.
2. Make sure that your file names do not have spaces in them. For example,
TechStack.txt or tech_stack.txt but not Tech Stack.txt as this will cause problem in reading this file using automated code if any such scenario arises.

File: docs/Tech Stack.txt [hunk 5]
```diff
@@ -1,11 +1,43 @@
...
+Selected: Spring Boot
+Other options considered : Django, Node
+
+Reason: 
+
+
+Parser
+-----------------
+Selected: Java
+Other options considered : Java, Python
```
Comment (krvikas1011) [inline docs/Tech Stack.txt]: Other options cannot have Java right?

File: docs/Tech Stack.txt [hunk 6]
```diff
@@ -1,11 +1,43 @@
...
+
+Reason:
+
+
+GitHub handler
+-----------------
+Selected: Java
+Other options considered : Java
```
Comment (krvikas1011) [inline docs/Tech Stack.txt]: Same here also

File: docs/Tech Stack.txt [hunk 7]
```diff
@@ -1,11 +1,43 @@
...
+
+Reason:
+
+
+UI
+-----------------
+Selected: React
+Other options considered : React, AngularJS
```
Comment (krvikas1011) [inline docs/Tech Stack.txt]: Same

**Review by thisissvikas — CHANGES_REQUESTED** [review]

File: docs/Tech Stack.txt (hunk 6 above)
Comment (krvikas1011) [reply docs/Tech Stack.txt]: add python here

**Review by thisissvikas — APPROVED** [review]

**Pull request merged**